clusters/app.ci/prow-monitoring/mixins/_prometheus/dptp_alerts.libsonnet

```diff
@@ -88,7 +88,7 @@
               severity: 'critical',
             },
             annotations: {
-              message: 'An excessive amount of CI Operator executions are failing with {{ $labels.reason }}, which is an infrastructure issue.',
+              message: 'An excessive amount of CI Operator executions are failing with `{{ $labels.reason }}`, which is an infrastructure issue. See <https://search.ci.openshift.org/?search=Reporting+job+state.*with+reason.*{{ $labels.reason }}&maxAge=6h&context=1&type=build-log&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job|CI search>.',
```

Comment:
This link doesn't match the link you shared in `#ops-testplatform` today. Do we want the  `Reporting+job+state.*with+reason.*` in here? Link for reference: https://search.ci.openshift.org/?search=loading_args%3Aloading_config%3Aconfig_resolver&maxAge=48h&context=1&type=build-log&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job

Reply:
The one I posted there was just the result of me pasting the reason into the search field.

For the alert, I'm being extra careful to avoid matching other messages that might happen to contain the reason string, as not all of them are this specific.  They give similar results:

- https://search.ci.openshift.org/?search=loading_args%3Aloading_config%3Aconfig_resolver&maxAge=48h&context=1&type=build-log&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job
- https://search.ci.openshift.org/?search=Reporting+job+state.*with+reason.*loading_args%3Aloading_config%3Aconfig_resolver&maxAge=48h&context=1&type=build-log&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job

I also used 48h to give a bit more context, it's always possible to adjust the period in the results page.